src/main/java/racingcar/domain/car/Cars.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.domain.car;
+
+import racingcar.strategy.MoveStrategy;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = new ArrayList<>(cars);
+    }
+
+    public List<String> getMaxPositionCarNames() {
+        int maxPosition = cars.stream()
+                .max(Comparator.comparing(Car::getPosition))
+                .orElseThrow()
+                .getPosition()
+                .getPosition();
+
+        return cars.stream()
+                .filter(car -> car.isWinnerPosition(maxPosition))
+                .map(car -> car.getName().getName())
+                .collect(Collectors.toList());
+    }
+
+    public Cars moveEachCar(MoveStrategy moveStrategy) {
+        List<Car> cars = new ArrayList<>();
+        this.cars.forEach(car -> cars.add(car.move(moveStrategy)));
+        return new Cars(cars);
+    }
+
+    public Collection<Car> getCollection() {
+        return cars;
+    }
```

Comment:
`getCollection()`의 의도는 무엇이었나요?
만약, `List<Car>`를 주는건 뭔가 좀 위험할것 같고 마음이 썩 좋지 않다 였다면, 

```suggestion
    public List<Car> getCars() {
        return Collections.unmodifiableList(cars);
    }
```

이렇게도 만들수 있습니다.